docs/dom.md

```diff
@@ -4,16 +4,22 @@ title: Working with DOM
 
 ## ReactDOM
 
-ReasonReact's ReactDOM module is called `ReactDOMRe`. The module exposes helpers that work with familiar ReactJS idioms.
+ReasonReact's ReactDOM module is called `ReactDOM`. The module exposes helpers that work with familiar ReactJS idioms
 
-- `render` : `(React.element, Dom.element) => unit`
-- `unmountComponentAtNode` : `Dom.element => unit`
-- `hydrate` : `(React.element, Dom.element) => unit`
-- `createPortal` : `(React.element, Dom.element) => React.element`
+- `ReactDOM.querySelector` : `string => option(Dom.element)`
+- `ReactDOM.Client.createRoot` : `Dom.element => Client.root`
+- `ReactDOM.Client.render` : `(Client.root, React.element) => unit`
+- `ReactDOM.Client.hydrateRoot` : `(Dom.element, React.element) => Client.root`
+- `ReactDOM.createPortal` : `(React.element, Dom.element) => React.element`
+- `ReactDOM.unmountComponentAtNode` : `Dom.element => unit`
+
+More info about `ReactDOM` can be found in the [official ReactDOM documentation](https://react.dev/reference/react-dom).
 
 ## ReactDOMServer
 
-ReasonReact's equivalent `ReactDOMServer` exposes:
+ReasonReact's equivalent of `ReactDOMServer` from `react-dom/server` exposes
 
 - `renderToString` : `React.element => string`
 - `renderToStaticMarkup` : `React.element => string`
+
+More info about `ReactDOMServer` can be found in the [official React documentation](https://react.dev/reference/react-dom/server).
```

Comment:
we should add a small section about `ReactDOMServerNode`, which you can get by depending on `reason-react.node`